File: AkitaModel/Results/Panel.cs
using System.Collections.Generic;

namespace Skyware.Lis.AkitaModel.Results;

/// <summary>
/// Represents a panel of tests (Profile).
/// </summary>
public class Panel : ExaminationBase
{

    /// <inheritdoc />
    public override bool IsPanel => true;

    /// <summary>
    /// Test items that are part of the panel.
    /// </summary>
    public IList<Test> Tests { get; set; }

}

File: AkitaMotelTS/Results/Panel.ts
import { ExaminationBase } from "./ExaminationBase";
import { Test } from "./Test";

namespace Skyware.Lis.AkitaModel.Results {
  /**
   * Represents a panel of tests (Profile).
   */
  export class Panel extends ExaminationBase {
    /**
     * If the examination is a panel, this property will be true.
     */
    IsPanel: boolean = true;

    /**
     * Test items that are part of the panel.
     */
    Tests: Test[] = [];
  }
}
